prow/plugins/trigger/generic-comment.go

```diff
@@ -156,13 +157,16 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 						"repo":    repo,
 					})
 					runID := run.ID
-					go func() {
```

Comment:
Is there a reason we aren't running this in a separate goroutine anymore?

Reply:
I was not sure if we want to Run the config.Presubmit in the case of expired workflow. Also I referred to this https://github.com/kubernetes/test-infra/blob/3aef270d485c0f747c9c5e310ef6e77d4ea5ba85/prow/plugins/trigger/generic-comment.go#L95 but not sure if that case applies to this too.